### ISSUE TITLE
Example of Store data in Azure devops extension

### ISSUE THREAD

**Issue by wqef3pks1** [MicrosoftDocs/azure-devops-docs#13097]
Hi It would be nice to have a short real example of the use of each aspect. For example in store data. 

What Im trying to do is a simple html page with 2 inputs, 1 button and store in azure devops extensión, but I cannot with the info  provided in documentation. 

// this is my TS file
const Commited1 = document.querySelector<HTMLInputElement>("#Commited1");
const Completed1 = document.querySelector<HTMLInputElement>("#Completed1");
const form = document.getElementById("form-new") as HTMLFormElement | null;

form?.addEventListener("submit", (e) => {
  e.preventDefault();

  if (Commited1?.value == "" || Commited1?.value == null||Completed1?.value == "" || Completed1?.value == null) {
    return;
  }

      // Get data service
      VSS.getService(VSS.ServiceIds.ExtensionData).then(function(dataService) {
        // Prepare document first
        var newDoc = {
            Commited1.value,
            Completed1.value
            
        };

        dataService.createDocument("MyCollection", newDoc).then(function(doc) {
            // Even if no ID was passed to createDocument, one gets generated
            console.log("Doc id: " + doc.id);
        });
    });

})

---------
The problem I suppose should be in a) the import, because in VSS say: "Cannot find name 'VSS'.ts(2304)". B) in newDoc.  

Can u help me and if u can post the example for evreyone could be great

Its my first time in rising the Question so I hope I did well.

**Comment by WilliamAntonRohm** [comment]
@wqef3pks1 -- thank you for your feedback.  This a great first issue.  Are you following a particular learn.microsoft.com article?  Meanwhile, please consider these resources:
- [Azure DevOps Services](https://azure.microsoft.com/support/devops/) 
- [Azure DevOps on Stack Overflow](https://stackoverflow.com/questions/tagged/azure-devops)

**Comment by wqef3pks1** [comment]
Yes  the article is: https://learn.microsoft.com/en-us/azure/devops/extend/develop/data-storage?toc=%2Fazure%2Fdevops%2Fmarketplace-extensibility%2Ftoc.json&view=azure-devops

**Comment by WilliamAntonRohm** [comment]
@wqef3pks1 -- thank you for your followup.

@chcomley -- please look into this issue.
